src/modules/common/models/TileModel.js

```diff
@@ -47,4 +53,12 @@ export default class TileModel {
   get postData (): ?Map<string, string> {
     return this._postData
   }
+
+  get onTilePress (): ?() => void {
```

Comment:
I just extended the existing `TileModel`s, do you think it's better to create individual Models?